posts/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+
+from core.models  import TimeStampModel
+from users.models import User
+
+class Location(models.Model):
+    address   = models.CharField(max_length=255)
+    latitude  = models.DecimalField(max_digits=10, decimal_places=6)
+    longitude = models.DecimalField(max_digits=10, decimal_places=6)
+
+    class Meta:
+        db_table = 'locations'
+
+class Category(models.Model):
+    category_name = models.CharField(max_length=20)
```

Comment:
```suggestion
    name = models.CharField(max_length=20)
```
views.py 에서 코드가 작성된다고 했을때를 고려해보면 쉽게 이해가 되실 것 같습니다.
```python
. . .

category = Category.objects.get(id=1)

. . . 

category_data = {
    'id' : category.id,
    #'name' : category.category_name,
    'name' : category.name
}

Reply:
수정하였습니다.